type-definitions/ts-tests/functional.ts

```diff
@@ -1,84 +1,63 @@
-import {
-  get,
-  has,
-  set,
-  remove,
-  update,
-} from 'immutable';
+import { expect, test } from 'tstyche';
+import { get, has, set, remove, update } from 'immutable';
 
-{
-  // get
+test('get', () => {
+  expect(get([1, 2, 3], 0)).type.toEqual<number | undefined>();
 
-  // $ExpectType number | undefined
-  get([1, 2, 3], 0);
+  expect(get([1, 2, 3], 0, 'a')).type.toEqual<number | 'a'>();
 
-  // $ExpectType number | "a"
-  get([1, 2, 3], 0, 'a');
+  expect(get({ x: 10, y: 20 }, 'x')).type.toEqual<number | undefined>();
 
-  // $ExpectType number | undefined
-  get({ x: 10, y: 20 }, 'x');
+  expect(get({ x: 10, y: 20 }, 'z', 'missing')).type.toEqual<
+    number | 'missing'
+  >();
+});
 
-  // $ExpectType number | "missing"
-  get({ x: 10, y: 20 }, 'z', 'missing');
-}
+test('has', () => {
+  expect(has([1, 2, 3], 0)).type.toBeBoolean();
 
-{
-  // has
+  expect(has({ x: 10, y: 20 }, 'x')).type.toBeBoolean();
+});
 
-  // $ExpectType boolean
-  has([1, 2, 3], 0);
+test('set', () => {
+  expect(set([1, 2, 3], 0, 10)).type.toEqual<number[]>();
 
-  // $ExpectType boolean
-  has({ x: 10, y: 20 }, 'x');
-}
+  expect(set([1, 2, 3], 0, 'a')).type.toRaiseError();
```

Comment:
Here and elsewhere `.not.toBeCallableWith()` matcher should be used instead of `.toRaiseError()`. Keeping it as it was , but I will come back after implementing `.toBeCallableWith()`:

```ts
expect(set).type.not.toBeCallableWith([1, 2, 3], 0, 'a');
```